Ascend.java

```diff
@@ -3,5 +3,24 @@
 public class Ascend {

```

Comment:
OVERALL:
This is an ok submission, 
Logic wise, you have issue in TimeFormat.java please see my comment there, you could have also write Ascend better.
Styling wise there is a lot to improve. please go over styling guidelines